index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Projects</title>
+
+    <style>
+      body {
+        padding: 0;
+        font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+      }
+
+      .flexbox-row {
+        display: flex;
+        gap: 10px;
+        flex-wrap: wrap;
+      }
+
+      .flexbox-col {
+        display: flex;
+        flex-direction: column;
+        padding-left: 10px;
+      }
+
+      .content-center{
+        justify-content: center;
+      }
+
+      .items-center{
+        align-items: center;
+      }
+
+      .button {
+        padding: 4px 8px;
+        box-shadow: 0 5px rgb(99, 98, 98);
+        background-color: rgb(155, 155, 155);
+        border-radius: 10px;
+      } 
+
+      .button:hover {
+        margin-top: 2px;
+        cursor: pointer;
+        box-shadow: 0 3px rgb(99, 98, 98);
+        margin-bottom: -2px;
+      }
+
+      .button:active {
+        margin-top: 3px;
+        cursor: progress;
+        box-shadow: 0 2px rgb(99, 98, 98);
+        margin-bottom: -3px;
+      }
+
+      div a {
+        text-decoration: none;
+        color: aliceblue;
+        font-weight: bold;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="flexbox-row content-center items-center">
+      <div class="flexbox-col" >
+        <h1>HTML and CSS</h1>
+      <div class= "flexbox-row" style="max-width: 800px">
```

Comment:
Indentation for this block should be increased by 2 spaces

Reply:
done